lib/screens/login_page.dart

```diff
@@ -105,116 +85,272 @@ class _LoginPageState extends State<LoginPage> {
-            Padding(
-              padding: const EdgeInsets.symmetric(vertical: 16.0),
-              child: ElevatedButton(
-                onPressed: () async {
-                  Navigator.push(
-                    context,
-                    MaterialPageRoute(
-                      builder: (context) => SignupPage(),
-                    ),
-                  );
-                },
-                child: Text('signup'),
-              ),
+          ),
+          Text(
+            'OR SIGN IN WITH',
+            style: TextStyle(color: Colors.black45),
+          ),
+          Expanded(
+            child: Padding(
+              padding: EdgeInsets.fromLTRB(10.0, 1.0, 10.0, 1.0),
+              child: Divider(color: Colors.black),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Container _forgotPasswordButton(context) {
+    return Container(
+      alignment: Alignment.centerRight,
+      child: TextButton(
+        child: Text(
+          "Forgot password?",
+          style: TextStyle(color: Colors.grey),
+        ),
+        onPressed: () async {
+          Navigator.push(
+            context,
+            MaterialPageRoute(
+              builder: (context) => ForgotPassword(),
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  Container _loginButton(context) {
+    return Container(
+      width: 350.0,
+      height: 45.0,
+      child: RaisedButton(
+        onPressed: () async {
+          setState(() {
+            showSpinner = true;
+          });
+          validateAndSubmitEmail();
+        },
+        color: Color(0xFF00ADB5),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(20.0),
+        ),
+        child: Container(
+          padding: EdgeInsets.symmetric(
+            horizontal: 20.0,
+          ),
+          child: Text(
+            'Sign In',
+            style: TextStyle(
+              color: Colors.white,
+              fontSize: 20.0,
             ),
-            Padding(
-              padding: const EdgeInsets.symmetric(vertical: 16.0),
-              child: TextButton(
-                onPressed: () async {
-                  Navigator.push(
-                    context,
-                    MaterialPageRoute(
-                      builder: (context) => ForgotPassword(),
-                    ),
-                  );
-                },
-                child: Text('Forgot password?'),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Row _signUpButton(context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: <Widget>[
+        Text(
+          "Don't have an account?",
+          style: TextStyle(
+            color: Color(0xFF00ADB5),
+            fontWeight: FontWeight.w500,
+          ),
+        ),
+        TextButton(
+          child: Text(
+            "Sign up",
+            style: TextStyle(
+              color: Colors.black,
+            ),
+          ),
```

Comment:
I would switch the colors here to make the prompt black and SignUp button blue, as it will be more intuitive which one is a button. Same on Sign Up page